src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -78,6 +78,8 @@ def __init__(self, overrides=None, **kwargs):
             kwargs['options_list'] = [options_list]
         self.settings = {}
         self.update(overrides, **kwargs)
+        self.required = None
+        self.default_name = None
```

Comment:
Since these 2 fields are only used by VS Code, to ease people's code reading effort to work on the real arg parsing logics, I suggest:
1. Rename it with suffix like `required_tooling`
2. Remove them from the `__init__`, rather use `setattr` to set values in `_resolve_default_value_from_cfg_file`